akvo/rsr/views/py_reports/email_report.py

```diff
@@ -1,50 +0,0 @@
-import logging
-
-from datetime import timedelta
-from django.db.models import Q
-from django.utils.timezone import now
-from akvo.rsr.models import EmailReportJob
-
-from . import (
-    program_overview_pdf_report,
-    program_overview_excel_report,
-    program_period_labels_overview_pdf_report,
-    results_indicators_with_map_pdf_reports,
-    nuffic_country_level_map_report,
-)
-
-TIMEOUT = timedelta(minutes=30)
-MAX_ATTEMPTS = 3
-HANDLER = {
-    program_overview_pdf_report.REPORT_NAME: program_overview_pdf_report.handle_email_report,
-    program_overview_excel_report.REPORT_NAME: program_overview_excel_report.handle_email_report,
-    program_period_labels_overview_pdf_report.REPORT_NAME: program_period_labels_overview_pdf_report.handle_email_report,
-    results_indicators_with_map_pdf_reports.ORG_PROJECTS_REPORT_NAME: results_indicators_with_map_pdf_reports.handle_org_projects_email_report,
-    nuffic_country_level_map_report.REPORT_NAME: nuffic_country_level_map_report.handle_email_report,
-}
-
-logger = logging.getLogger(__name__)
-
-
-def run_job():
```

Comment:
The command `akvo/rsr/management/commands/send_report_via_email.py` won't work anymore. It can be deleted.

Reply:
Thanks!